mobilitypath_publisher/README.md

```diff
@@ -0,0 +1,3 @@
+# mobilitypath_publisher
+
+TODO for USER: Add description of package and link to confluence documentation.
```

Comment:
Add description of what this package does. And a link to the confluence design doc if it exists. If not, no need for the link.